src/keysym-utf.c

```diff
@@ -810,7 +810,7 @@ static const struct codepair keysymtab[] = {
     { 0x0ef0, 0x3171 }, /*    Hangul_SunkyeongeumMieum ㅱ HANGUL LETTER KAPYEOUNMIEUM */
     { 0x0ef1, 0x3178 }, /*    Hangul_SunkyeongeumPieub ㅸ HANGUL LETTER KAPYEOUNPIEUP */
     { 0x0ef2, 0x317f }, /*              Hangul_PanSios ㅿ HANGUL LETTER PANSIOS */
-/*  0x0ef3                  Hangul_KkogjiDalrinIeung ? ??? */
+    { 0x0ef3, 0x3181 }, /*    Hangul_KkogjiDalrinIeung ㆁ HANGUL LETTER YESIEUNG */
```

Comment:
Would be great if some familiar with Ancient Hangul Consonant Characters could confirm this. The [mgk25 pdf](https://www.cl.cam.ac.uk/~mgk25/ucs/keysyms.pdf) does not state where they were taken from and does not specify a Unicode equivalent for them, and searching for "Hangul Kkogji Dalrin Ieung" does not bring up something I am able to understand...

Reply:
OK, I found this was recently added in xorgproto: https://gitlab.freedesktop.org/xorg/proto/xorgproto/-/commit/959d4442e46d8d3bef845e526219d23346a1328e

I assume the proper research was done there, so I'll pick this up too.

Reply:
I agree. I just applied the character in the comment, I am no expert.